lib/service/commands/list.rbi

```diff
@@ -0,0 +1,5 @@
+# typed: strict
+
+module Service::Commands::List
+  include Kernel
+end
```

Comment:
If you do a single `service.rbi` that has

```rb
module Service
  include Kernel
end
```

it should apply to everything else.

We do something similar with the `Homebrew` module in Homebrew/brew.

Reply:
Hmm, apparently not, I tried that (`service.rbi` alongside `service.rb`) but it gave 38 errors all related to `Kernel` so I had to make individual RBIs?

Reply:
You can also do `include Kernel` in the actual modules.

Alternatively, I think in Homebrew/brew a `module.rbi` with the following might work:
```ruby
module Module
  include Kernel
end
```

Reply:
This is mostly caused by using `module_function` to be honest.

Should be able to remove all the RBIs when rebased on #714.